tests/Unit/Infrastructure/Processing/RedisAssetTerminalStatusCacheTest.php

```
@@ -0,0 +1,71 @@
+
+    #[Test]
+    public function itThrowsWhenTheCacheWriteFails(): void
+    {
+        // Arrange
+        $cache = new RedisAssetTerminalStatusCache(
+            static function (string $_key, string $_value, int $_ttl): bool {
+                self::assertIsString($_key);
+                self::assertIsString($_value);
+                self::assertIsInt($_ttl);
+
+                throw RedisAssetTerminalStatusCacheException::storeFailed();
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\App\Infrastructure\Processing\Exception\RedisAssetTerminalStatusCacheException' in method 'itThrowsWhenTheCacheWriteFails'. (undefined)

(StaticAccess)